Timeseries chart

danielfdsilva opened the pull request:
Review after https://github.com/openaq/openaq.org/pull/523. Had to merge from `feature/compare` otherwise the homepage wouldn't load.

Addresses the label issues on the Timeseries chart. Example:
![image](https://user-images.githubusercontent.com/1090606/103928602-fa453480-5113-11eb-9e50-c01d84213401.png)

![image](https://user-images.githubusercontent.com/1090606/103928621-03ce9c80-5114-11eb-9981-789ee9f228aa.png)

rwegener2 reviewed (changes requested):
* (y-axis of time series) I like that you added the name of the pollutant and the unit.  On my screen the unit gets cuts off.  Can we either add the unit in a new line or increase the height of the time series table?
<img width="561" alt="Screen Shot 2021-01-07 at 4 34 21 PM" src="https://user-images.githubusercontent.com/35503632/103952539-340f3e80-5106-11eb-95b9-97ad4db32065.png">

* (All labels) Can we make the text bigger?  They sort of blend in to me.
* For the y-axis label on Temporal Coverage, can we write "Count" instead of "Total"?  Because it is technically an average of counts I think the word total might be misleading.

Thank you!

danielfdsilva commented:
@rwegener2 Addresses feedback:
- It seems it is not possible to add multiple lines on axis labels, so i added the unit to the first vertical tick
- Font size increased
- Replaced "Total" with "Count"

rwegener2 reviewed (approved):
All updates look good.  The one issue I had was that the unit still shows up funny for me (Pushed into the number labels).  
<img width="727" alt="Screen Shot 2021-01-08 at 7 40 01 AM" src="https://user-images.githubusercontent.com/35503632/104021632-b9393880-5184-11eb-96e8-db829d2620b1.png">
It seems like this may take a little more investigating to figure out something that will look nice so let's merge as is for now (EDF will be happy we put on some kind of label) and come back to it next week.

jvntf commented on app/assets/scripts/components/line-chart.js:
```diff
@@ -20,31 +20,53 @@ export default function LineChart({ data, xUnit }) {
+            tooltipFormat: 'MMM D, YYYY',
             unit: xUnit || 'day',
+            displayFormats: {
+              day: 'MMM D, YYYY',
```
Perhaps this should be formatted as MM/DD/YY to save space? current formatting seems crowded and repetetive.

jvntf reviewed (changes requested):
+1 for the unit overlap, other than that looks great
